operations/helm/charts/mimir-distributed/templates/provisioner/provisioner-job.yaml

```diff
@@ -0,0 +1,119 @@
+{{- if and .Values.provisioner.enabled .Values.enterprise.enabled (not .Values.federation_frontend.disableOtherComponents) -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "mimir.resourceName" (dict "ctx" . "component" "provisioner") }}
+  labels:
+    {{- include "mimir.labels" (dict "ctx" . "component" "provisioner") | nindent 4 }}
+    {{- with .Values.provisioner.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- with .Values.provisioner.annotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+    "helm.sh/hook": {{ .Values.enterprise.provisioner.hookType | quote }}
+    "helm.sh/hook-weight": "20"
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  backoffLimit: 6
+  completions: 1
+  parallelism: 1
+  selector:
+  template:
+    metadata:
+      labels:
+        {{- include "mimir.podLabels" (dict "ctx" . "component" "provisioner") | nindent 8 }}
+      annotations:
+        {{- include "mimir.podAnnotations" (dict "ctx" . "component" "provisioner") | nindent 8 }}
+      namespace: {{ .Release.Namespace | quote }}
+    spec:
+      serviceAccountName: {{ include "mimir.resourceName" (dict "ctx" . "component" "provisioner") }}
+      {{- if .Values.provisioner.priorityClassName }}
+      priorityClassName: {{ .Values.provisioner.priorityClassName }}
+      {{- end }}
+      securityContext:
+        {{- include "mimir.lib.podSecurityContext" (dict "ctx" . "component" "provisioner") | nindent 8 }}
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end }}
+      {{- end }}
+      initContainers:
+        {{- range $index, $tenant := .Values.provisioner.additionalTenants }}
+        - name: provisioner-{{ $tenant.name }}
+          image: "{{ $.Values.provisioner.image.registry }}/{{ $.Values.provisioner.image.repository }}:{{ $.Values.provisioner.image.tag }}"
+          imagePullPolicy: {{ $.Values.provisioner.image.pullPolicy }}
+          command:
+            - /usr/bin/provisioner
+          args:
+            - -bootstrap-path=/bootstrap
+            - -cluster-name={{ include "mimir.clusterName" $ }}
+            - -api-url={{ $.Values.provisioner.apiUrl $ }}
```

Comment:
isn't this `cortex-url` instead of `api-url`?

Reply:
Nope.  It changed to product neutral terms because it's being used by all three databases now.  in Loki this was "gel-url".  You can see all the current supported CLI args at: https://github.com/grafana/enterprise-provisioner/blob/main/cmd/enterprise-provisioner/main.go#L29-L36